WordPress/src/main/java/org/wordpress/android/ui/JetpackConnectionSource.kt

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.ui
+
+enum class JetpackConnectionSource(private val value: String) {
```

Comment:
1) Suggestion (💡): I usually suggest to have the construction fields in a separate line instead. I usually prefer to do that for every class. However, for `data` and `enum` classes I am more intrigue in doing so. This is merely me trying to share what I usually do in such case, not insisting on you changing this to my style, just me trying to sync with you.
2) Suggestion (💡): However, for the actual enums themselves (`NOTIFICATIONS` and `STATS`, I would suggest we do keep them in separate lines. Wdyt?

Reply:
> I usually suggest to have the construction fields in a separate line instead. I usually prefer to do that for every class.

Reply:
> I believe the code base leans more towards constructor fields on the same line, especially when it's a single param.

> Again, the code base uses both syntax.

> Perhaps this might be a good discussion for our Android tech chat and see if we want to standardize on some of these things. Neither way is wrong, what reads easier for one person, may read harder for another. I bet we'll get 50 different opinions. :) At least let's bring it up in our next Raven sync and see if the 3 of us can come to an agreement.

Yes, you are right, I am going to add it to the backlog, thanks for the idea! 💡 

> Neither way is wrong, what reads easier for one person, may read harder for another.

Actually, in this case, at least version control wise, having those in separate lines is a better alternative and it is no longer that much subjective. If you get those on separate lines and then you choose to update something specific, then the diff will only show that specific update, where is actually makes sense. For example, if you update `NOTIFICATIONS("notifications")` to `NOTIF("notif")`, and have that logic in a separate line, then you wouldn't care about reading `STATS("stats")` in the diff. However, if those are on the same line then you are inevitable forced to review the `STATS("stats")` as well.

The same applies for having the constructor fields in separate line. Actually, the `trailing comma` feature of Kotlin 1.4 is useful because it is basing its usefulness on the same premise, to have as less diff on version control as possible.

> Thanks for sharing :)

Thank YOU! ❤️